### ISSUE TITLE
STAS Review and adjust the about search results page for accessibility

### ISSUE THREAD

**Issue by Captainkirkdawson** [FreeUKGen/MyopicVicar#965]

**Comment by stsccfr** [comment]
Closing as old and lacking a description to guide action
